### PR TITLE
jit-pmi-diff: disable tiering by default

### PR DESCRIPTION
**Pull request by AndyAyersMS**
Turn off tiering by default for when invoking PMI. Can enable via -tiering.

Also, fix a couple of spelling errors.

### CODE REVIEW

**Comment by AndyAyersMS** [comment]
@BruceForstall PTAL
cc @dotnet/jit-contrib

File: src/jit-dasm-pmi/jit-dasm-pmi.cs [hunk 1]
```diff
@@ -147,6 +149,7 @@ private void Validate()
         public IReadOnlyList<string> PlatformPaths { get { return _platformPaths; } }
         public string FileName { get { return _fileName; } }
         public IReadOnlyList<string> AssemblyList { get { return _assemblyList; } }
+        public bool Tiering => _tiering;
```
Comment (BruceForstall) [inline src/jit-dasm-pmi/jit-dasm-pmi.cs]: Doesn't this syntax make the property read/write, when all we really want is to make it readable?

Comment (AndyAyersMS) [reply src/jit-dasm-pmi/jit-dasm-pmi.cs]: No, this is just a getter, to be able to set you'd need something like:
```
   public Bool Tiering
   {
      get => _tiering;
      set => _tiering = value;
   }
```

**Comment by AndyAyersMS** [comment]
Tiering doesn't impact crossgen codegen, so jit-dasm doesn't care one way or the other... or are you suggesting something else?

**Comment by BruceForstall** [comment]
Oh, forgot about that. ok.

**Pull request merged**